docs/source/authentication.rst

```diff
@@ -7,32 +7,11 @@ That means that any authentication method supported by an installed PAS Plugin s
 
 For example, to authenticate using HTTP basic auth, you'd set an ``Authorization`` header:
 
-.. code-block:: http
-
-  GET /Plone HTTP/1.1
-  Authorization: Basic Zm9vYmFyOmZvb2Jhcgo=
-  Accept: application/json
+..  http:example:: curl httpie python-requests
+    :request: _json/siteroot.req
 
 HTTP client libraries usually contain helper functions to produce a proper ``Authorization`` header for you based on given credentials.
 
-Using the ``requests`` library, you'd set up a session with basic auth like this:
```

Comment:
@kakshay21 may I ask why you removed this section? (Sorry for not coming back to this sooner).

Reply:
It's already covered.
I build the documentation to show you.
![screen shot 2018-03-20 at 5 53 15 pm](https://user-images.githubusercontent.com/13598994/37654396-e544c560-2c67-11e8-9506-413062e7f25f.png)

Reply:
@tisto any update on this?